plugins/chat/app/controllers/chat/api/channels_memberships_controller.rb

```diff
@@ -44,4 +44,18 @@ def create
       end
     end
   end
+
+  def destroy
+    Chat::RemoveUserFromChannel.call(service_params) do
+      on_success { render(json: success_json) }
+      on_failure { render(json: failed_json, status: 422) }
+      on_model_not_found(:channel) { raise Discourse::NotFound }
```

Comment:
Should we have something similar for `:target_user`?

Reply:
Sure. Seems prudent. 🙂